examples/volumes/limits_and_quota/README.md

```diff
@@ -0,0 +1,53 @@
+# Limiting storage resource consumption
```

Comment:
doesn't this type of document (which is good) belong in the kubernetes.github.io repo instead?